### DISCUSSION TITLE
Extracting text from specified area

### DISCUSSION THREAD

**Discussion by ajaysurya1221** [py-pdf/pypdf#1418] [Q&A]
is there a way to extract text from pdf by specifying xmin, ymin, xmax, and ymax? I am trying to pass the bounding box coordinates detected from Yolo model to pypdf2.

**Comment by pubpub-zz** [comment]
The visitor function should provide you with the feature you are looking for:
https://pypdf2.readthedocs.io/en/latest/user/extract-text.html#using-a-visitor

**Comment by ajaysurya1221** [comment]
Thanks @pubpub-zz , I'll give it a try :)

**Reply by MartinThoma** [reply]
@ajaysurya1221 Did it work? Could you add some minimal code / add an example for the docs?
